quesma/quesma/dual_write_proxy_v2.go

```diff
@@ -110,24 +110,14 @@ func newDualWriteProxyV2(schemaLoader clickhouse.TableDiscovery, logManager *cli
 		return routerInstance.failedRequests.Load()
 	})
 
-	handler := http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		defer recovery.LogPanic()
-		reqBody, err := peekBodyV2(req)
-		if err != nil {
-			http.Error(w, "Error reading request body", http.StatusInternalServerError)
-			return
-		}
-
-		ua := req.Header.Get("User-Agent")
-		agent.UserAgentCounters().Add(ua, 1)
+	elasticHttpFrontentConnector := NewElasticHttpFrontendConnector(":"+strconv.Itoa(int(config.PublicTcpPort)),
```

Comment:
Introduction of `elasticHttpFrontendConnector`, finally there will be 2 (one for ingest and one for query)